app/models/Country.scala

```diff
@@ -22,7 +22,7 @@ case class Country(state: String, code: String, description: String, alternative
 
 object Country {
 
-  val GB = Country("valid", "GB", "United Kingdom")
```

Comment:
Shall we remove the 'state' parameter? - it's not used anywhere else, this appears to be the last on only occurrence